src/main/java/somethingrandom/entity/DelayedItem.java

```diff
@@ -9,35 +9,39 @@
 public class DelayedItem extends Item {
     private String name;
```

Comment:
I think these are unused now, and they're begging to be misused, so they probably should be removed.